classes/UI/Console/console.js

```diff
@@ -266,7 +266,7 @@ class Console {
                 title: json['title'],
                 description: json['description'],
                 channel: guild.channels.resolve(json['channelId']),
-                features: new Collection(json['features']),
+                features: new Collection(Feature.create(json['features'])),
```

Comment:
I think the jason['features'] has mutliple features, you will need to loop over each one and create a Feature object for each!

Reply:
It'll just be like `json['features'].forEach(feature => Feature.create(feature)) `?

Reply:
I think that should work, you should try it tho!